### ISSUE TITLE
Hotkey activation

### ISSUE THREAD

**Issue by webmastak** [sunwxg/gnome-shell-extension-dict#1]
Is it possible to make a choice to activate the translation by a hot key and turn off the icon or by a hot key and icon?

**Comment by sunwxg** [comment]
Do you means to use hot key to disable and enable the extension?

**Comment by webmastak** [comment]
I want to use the hot key to activate the translation of the selected text.

**Comment by sunwxg** [comment]
Okay, I get it. Which hot key do you prefer to do this work?

**Comment by webmastak** [comment]
It is best to choose which keys to turn on.

**Comment by sunwxg** [comment]
I think It's better to use same key to trigger on and off. Press once to turn on, then press again to turn off.

**Comment by webmastak** [comment]
> I think It's better to use same key to trigger on and off. Press once to turn on, then press again to turn off.

All right.

But you me did not understand) Make it so that can set the hotkey which one wants to use.

**Comment by sunwxg** [comment]
It's a little difficult to set hotkey manually. I prefer to use the fix hotkey, or select one hotkey from several hotkeys.

**Comment by sunwxg** [comment]
You can try my another extension 'Switch Workspace'. In this extension, user can chose the hotkey from  4 hotkeys.

**Comment by webmastak** [comment]
In this [extension](https://gitlab.com/LinxGem33/Arc-Menu) for example, in the settings you can set the hot key yourself. 

I'm not strong in the java script unfortunately,  I would have done it myself, may be rummaging at leisure. ))

**Comment by sunwxg** [comment]
Yes, in that extension we can define hotkey manually. You can see the code in prefs.js, almost 90% codes are for manually defining hotkeys. And you should consider many other things, one important is conflict with other hotkeys. If you can't consider all things, you will meet many bugs. So I prefer to use fix hotkey.

**Comment by webmastak** [comment]
You can simply write hotkeys to a constant? For example `const HOTKEY = "<Super>z";`
Who will need to change, he will climb and fix in the file.

**Comment by sunwxg** [comment]
Yes, I can. If we use fix hotkey, we will define it in the code as that.

**Comment by webmastak** [comment]
Good

**Comment by sunwxg** [comment]
New feature has been finished. You can get it from extensions.gnome.org . Now the default hotkey is Ctrl+Alt+j .
If you want to change hotkey, do as follow:
```
gsettings --schemadir ~/.local/share/gnome-shell/extensions/dict@sun.wxg@gmail.com/schemas get org.gnome.shell.extensions.dict hotkey
gsettings --schemadir ~/.local/share/gnome-shell/extensions/dict@sun.wxg@gmail.com/schemas set org.gnome.shell.extensions.dict hotkey "<Control><Shift>j"
```

**Comment by webmastak** [comment]
Thank

**Comment by webmastak** [comment]
But you still do not understand me)))
Hotkeys do not enable / disable the extension.
Hot keys need to activate the window translation.
And when you activate text translation with hotkeys, disable the pop-up icon.

And when you add your web address to the translation, Google for example. An empty window opens!
`https://translate.google.ru/?sl=auto&tl=ru&text=%WORD`

So the perfect solution for me remains - [Text Translator](https://extensions.gnome.org/extension/1756/text-translator/)
